scio-core/src/main/scala/com/spotify/scio/ScioContext.scala

```diff
@@ -653,13 +653,43 @@ class ScioContext private[scio] (
   /**
    * Runs the underlying pipeline according to the options used to create this context.
    * Running closes the context and no operation can be performed once the context is closed.
+   *
+   * This operation is non-blocking and no operation can be performed once the context is closed.
```

Comment:
"according to the options used to create this context." is a bit redundant and implied?

Clarify `no further operation`? No further transformations?